src/misc/msccl/msccl_lifecycle.cc

```diff
@@ -592,6 +592,7 @@ ncclResult_t mscclEnqueueCheck(
         }
       threadLocalStatus.groupStatus = mscclGroupUnsupportedOp;
       NCCLCHECK(mscclFallBackSavedParams());
+      break;
     case mscclGroupUnsupportedOp:
```

Comment:
Alternatively, is this fall-through intentional? If so, it double calls `mscclFallBackSavedParams`. Is that also intentional?